content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+
+JCasC defines the controller configuration in a YAML file.
+YAML is a popular serialization language for configuration information,
+with a syntax that is straight-forward and easy to read but precise.
+
+Some key points about YAML syntax:
+
+* YAML files are case sensitive.
+* Indentation is very significant and specific.
+* Each item is a key/value pair.
+** The key is followed by a colon (`:`) and a space.
+** YAML converts certain strings into other types unless they are in quotes.
+*** Values such as `true`, `false`, `Yes`, and `No` are converted to Boolean values.
+*** Values such as `2` and `3.0` are converted to floating point values.
+* A value can be a list:
+** Each list item is on a separate line starting with a dash (`-`).
+** Each list item in the file must start at the same indentation.
+*** Use spaces, never tabs, for indentation.
+** Never leave a blank line in a YAML file -- things will break!
+
+See the
+link:https://yaml.org/refcard.html[YAML Reference Card]
+for more details about YAML file syntax.
+
+== Checking the *.yaml File into SCM
+
+To get the maximum benefit of JCasC, the YAML files should be stored in SCM.
+This gives you a history that you can use to trace changes that are made
+and allow you to easily roll back to an earlier version of the file if necessary.
+
+JCasC does not require that the file be stored in SCM
+and so does not enforce any rules about how you do this.
+The most common practice is to create one SCM repository
+in which you store all of your JCasC files.
+
+If you are storing your JCasC YAML files in SCM,
+you should commit the first default file that is generated,
+before you make any modifications to the file.
+
+== Modifying the JCasC File
+
+To modify the JCasC YAML file,
+use the text editor of your choice to edit the file
+that is listed on the *Manage Jenkins >> Configuration as Code* UI page.
+By default, this is _$JENKINS_HOME/jenkins.yaml_.
+
+For a simple exercise to work through the process,
+you can modify the value of the "System Message"
+that is displayed on the Jenkins dashboard.
+
+. Open the JCasC YAML file with the text editor of your choice.
+. Find the `systemMessage` line near the top of the file:
++
+[source,yaml]
+----
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code plugin\n\n"
+----
+
+. Modify the text between the quotation marks to contain your new text
+. Write/save the file
+. Click the *Reload existing configuration* button to apply the changes
+. View the modified "System Message" on your dashboard
+
+[NOTE]
+====
+It is not necessary to restart Jenkins to apply the JCasC changes,
+although you should try to restart Jenkins with the modifications
+before you check the modified YAML file into SCM,
+especially when making more substantive configuration changes.
+====
+
+When you have made and tested your desired changes,
+push the modified JCasC YAML file to your SCM.
+
+== Configuring a Plugin with JCasC
```

Comment:
The [style guide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments) requests sentence case the "titles" (I interpret that to mean "headings", but that is not a consistent usage throughout the documents, for sure) (optional)

```suggestion
== Configuring a plugin with JCasC
```